apps/server/src/modules/tldraw/repo/tldraw.repo.ts

```diff
@@ -1,20 +1,59 @@
 import { EntityManager } from '@mikro-orm/mongodb';
 import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { BulkWriteResult, Collection, Sort } from 'mongodb';
+import { TldrawConfig } from '../config';
 import { TldrawDrawing } from '../entities';
 
 @Injectable()
 export class TldrawRepo {
-	constructor(private readonly _em: EntityManager) {}
+	private readonly collectionName: string;
 
-	async create(entity: TldrawDrawing): Promise<void> {
+	constructor(private readonly configService: ConfigService<TldrawConfig, true>, private readonly _em: EntityManager) {
+		this.collectionName = this.configService.get<string>('TLDRAW_DB_COLLECTION_NAME') ?? 'drawings';
```

Comment:
I find it to much when we make the collection name configurable. If we want to change it, than we must also write a migration script to move the existing data. Therefore it is fine if it must also change in code. The collection name should be added to the entity for the microORM logic.

Reply:
you are right, there is no point in having this configurable, I am removing this from config and instead using microOrm entity table name